pytest_pgsql/plugin.py

```diff
@@ -79,12 +79,7 @@ def database_uri(request):
             # SQLAlchemy when the driver isn't found.
             raise ImportError("Can't find the database driver %r." % driver)
 
-        # Break the connection URL into its parts, replace the protocol part
-        # ("postgresql") to define the driver (e.g. "postgresql+pygresql") and
-        # return that to the user.
-        url_parts = list(urllib.parse.urlparse(pgdb.url()))
-        url_parts[0] = 'postgresql+' + driver
-        yield urllib.parse.urlunparse(url_parts)
+        yield pgdb.url()
```

Comment:
@dargueta the custom URL construction based on the driver is not compatible with `psql`. At least trying to access this URL with `psql` during tests (programmatically or manually) does not work. There are plenty of cases for tests that need to shell out `psql` commands with the db url, so I think it'd be best to change it back

Reply:
Wait why are we using `psql`?

Reply:
There are cases for doing `subprocess.run` with the database URL on scripts that use psql